db/PGseeder.js

```diff
@@ -2,7 +2,18 @@ const faker = require('faker');
 const fs = require('fs');
 
 
-let count = 0;
+const seed = () => {
+  let count = 0;
+
+  while (count < 10) {
+      let data = '';
+      data += product;
+      count++;
+      fs.appendFileSync('./allTheData.tsv', data);
+  }
+  console.log(count);
+};
+
```

Comment:
for large data, the most efficient way i tested so far is a while loop combine with a for loop, ask roger for this question, but i think i could still do more optimize for it , try to pressure test with 1M data Gloria

Reply:
ok ty i will work on that now.

hmmmm i thought i deleted the tsv -- it wouldn't let me even push to my branch bc the file was too big. it's gone now. thank you!